tests/suites/test_suite_x509parse.data

```diff
@@ -735,6 +735,26 @@ X509 Certificate verification #87 (Expired CA and invalid CA)
 depends_on:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED:MBEDTLS_ECP_DP_SECP384R1_ENABLED:MBEDTLS_SHA1_C:MBEDTLS_SHA256_C
 x509_verify:"data_files/server5.crt":"data_files/test-ca2_cat-past-invalid.crt":"data_files/crl-ec-sha1.pem":"NULL":MBEDTLS_ERR_X509_CERT_VERIFY_FAILED:MBEDTLS_X509_BADCERT_EXPIRED:"compat":"NULL"
 
+X509 Certificate verification #88 (Suite B invalid, EC cert, RSA CA)
```

Comment:
Could you uniformize the naming scheme here? Suggestion: `"PROFILE_NAME" profile, [In]valid cert, additional information`. E.g. `"Suite B" Profile, Invalid EC Cert, RSA CA` or `"next" Profile, Invalid Cert, SHA-224 digest`.

Reply:
@hanno-arm this means modifying all the 92 x509_verify tests.
Very well, I'll do it tomorow

Reply:
@RonEld Ok, then let's put this aside as a potential future cleanup of the entire test suite.

Reply:
@RonEld Could you then only adapt pick a uniform capitalization of `invalid/valid`, and write `SHA-224 / SHA-256` instead of `SHA224 / SHA256`?